docs/formatter.qmd

```diff
@@ -173,6 +174,48 @@ list(foo, bar)
 
 The goal of this feature is to strike a balance between being opinionated and recognizing that users often know when taking up more vertical space results in more readable output.
 
+# Disabling formatting
+
+Air supports two special comments to disable formatting, `# fmt: skip` and `# fmt: skip file`.
+
+`# fmt: skip` skips formatting for the following syntax node (including all of its children).
+
+``` r
+# This skips formatting for `list()` and its arguments, retaining the manual alignment
+# fmt: skip
+list(
+  dollar = "USA",
+  yen    = "Japan",
+  yuan   = "China"
+)
+
+# This skips formatting for `tribble()` and its arguments
+# fmt: skip
+tribble(
+  ~x, ~y,
+   1,  2,
+)
+```
+
+`# fmt: skip file` skips the entire file.
+This must be placed at the top of the file before the first non-comment R code, otherwise the comment is ignored.
+Use this for generated files, or as an alternative to `exclude` for individual files that you know you'd like to exclude from formatting.
```

Comment:
Very happy to see this land in `air`! I was just taking a look at how this is implemented and noticed something unclear to me in the docs.

Is the sentence about generated files actually useful for end users? It seems to me that people who develop the framework should include this in the generated files, but end users shouldn't since it will be overwritten next time the file is generated. The example following this paragraph is also a bit confusing in this regard.